server/midddleware/metrics_test.go

```diff
@@ -33,39 +34,40 @@ func TestGrpcMetrics(t *testing.T) {
 	}
 	fullMethodName := "/tigrisdata.v1.Tigris/TestMethod"
 
-	metrics.InitServerRequestMetrics(svcName, methodInfo)
+	if config.DefaultConfig.Metrics.GrpcRequestCounters {
```

Comment:
Don't we want test it unconditionally?

Reply:
No, because the initialisation has the condition as well. If the initial structures are not created in tally, the test will panic. I will improve the tests to test the case when it's turned off as well.

Reply:
We can just set `config.DefaultConfig.Metrics.GrpcRequestCounters = true` in the beginning of the test?

Reply:
It's the same panic. I made the indentation nicer with checking this first.

Reply:
Scratch that, it works.